test/feature/step_definitions/setup_definitions.coffee

```diff
@@ -1,17 +1,26 @@
 { expect } = require 'chai'
+coffee = require 'coffee-script'
 
 
 module.exports = ->
 
   @Given /^an exprestive app using defaults$/, (done) ->
-    @createExprestiveApp '', (err) =>
+    @createExprestiveApp library: 'connect', (err) =>
+      return done err if err
```

Comment:
All these ugly and redundant error checks in tests! How do you feel about me adding some [Nitroglycerin](https://github.com/Originate/nitroglycerin) here?